images/base/Dockerfile

```diff
@@ -42,8 +44,6 @@ COPY --chmod=0644 files/etc/systemd/system/kubelet.service.d/* /etc/systemd/syst
 # Install dependencies, first from apt, then from release tarballs.
 # NOTE: we use one RUN to minimize layers.
```

Comment:
This comment is out of date (we squash with multi-stage) and I'd probably move the chmod to it's own RUN, but those cleanups aren't important and I want to get this into the next release and we have some other bug fixes to prompt a quick follow-up release currently